korge/src/commonMain/kotlin/com/soywiz/korge/input/SwipeComponent.kt

```diff
@@ -0,0 +1,139 @@
+package com.soywiz.korge.input
+
+import com.soywiz.korge.view.*
+import com.soywiz.korma.geom.*
+import kotlin.math.*
+
+data class SwipeInfo(
+    var dx: Double = 0.0,
```

Comment:
This is mutable, but then you construct for every event.
To be consistent with the rest of the events, this might be mutable indeed but you should construct an instance for the `onSwipe` block and trigger that instance these fields updated